Automattic/wp-calypso
# My Home: Add dot pager control to learn-grow section on My Home

#### Changes proposed in this Pull Request

* Add `<DotPager>` component to match My Home style
* Use standard Gutenberg icons in `<DotPager>` (as mentioned in #54226)
* Wrap learn-grow cards in a dot pager control
* Styling to make it look the same as before
* Show dot controls inside the card (the mockup in #54227 shows the controls outside, but follow up discussion in #54367 decided to put them inside)


https://user-images.githubusercontent.com/1500769/125759457-bf22c28c-3643-4945-a3bd-4a4731005e38.mov



#### Testing instructions



* Test various views to see how the new learn-grow section looks
	* `?view=VIEW_GO_MOBILE` for view with only one learn-grow card
	* `?view=VIEW_DEFAULT` for view with multiple learn-grow cards
	* `?view=VIEW_WP_FOR_TEAMS` for view with no learn-grow cards
* Test different screen sizes
* I've changed the `<DotPager>` control (of which My Home is the only user) so make sure it still works as a standalone: http://calypso.localhost:3000/devdocs/design/dot-pager



Related to #54227 (and #54226)
